xml/docker_building_images.xml

```diff
@@ -192,25 +174,28 @@ RUN zypper ref -s
 RUN zypper -n in vim</screen>
   </sect2>
 
-  <sect2 xml:id="sec.docker.sle_images.customizing_the_images.sles11sp3">
-   <title>Creating a Custom &slea;&nbsp;11 SP3 Image</title>
+<sect2 xml:id="sec.docker.sle_images.customizing_the_images.sles15">
+   <title>Creating a Custom &slea;&nbsp;15 Image</title>
    <para>
     The following Docker file creates a simple Docker image based on
-    &slea;&nbsp;11 SP4:
+    &slea;&nbsp;15:
    </para>
-<screen>FROM suse/sles11sp3:latest
+<screen>FROM registry.suse.com/suse/sles15
```

Comment:
Needs to be `registry.suse.com/suse/sle15`

Reply:
Thanks for your eagle eyes. :) Fixed in two places.